guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>MMdetection is an open-source library containing many popular and state-of-the-art object detection models. Through <code>arcgis.learn</code>, we have provided a bridge to use the growing list of models provided by the MMDetection library. </p><p>In 2018, the MMdet team won the COCO object detection challenge. Their codebase, which is built with Pytorch, has gradually evolved to include various models and methods. Although the detection task can be complex, the creators of MMDetection decomposed the models into different general components - Backbone, Neck, DenseHead, ROIExtractor, and ROIHead. With this abstraction, the library is able to provide a multitude of single-stage (e.g. GHM, FCOS), two-stage (e.g. Double-Head R-CNN), multi-stage (e.g. Cascade R-CNN), and other detection models. This abstraction-based approach also allows the development of models either by changing the different components or even by adding new ones. Read the MMDetection <a href="https://arxiv.org/pdf/1906.07155.pdf" target="_blank">paper</a>, or checkout its <a href="https://github.com/open-mmlab/mmdetection" target="_blank">github repository</a>, for more details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>